public/app/features/dashboard/dashgrid/PanelChromeAngular.tsx

```diff
@@ -173,12 +173,7 @@ export class PanelChromeAngularUnconnected extends PureComponent<Props, State> {
 
   hasOverlayHeader() {
     const { panel } = this.props;
-    const { errorMessage, data } = this.state;
```

Comment:
@torkelo if we move the corner out of the title element, we don't need this check